webapp/app/vitest.config.ts

```diff
@@ -12,7 +12,11 @@ export default mergeConfig(
       root: fileURLToPath(new URL('./', import.meta.url)),
       transformMode: {
         web: [/\.[jt]sx$/]
-      }
+      },
+      coverage: {
+        provider: 'istanbul',
+        reporter: ['text', 'json', 'html'],
```

Comment:
https://istanbul.js.org/docs/advanced/alternative-reporters/#json-summary

this reporter makes it easy to figure out what is the percentage of coverage